packages/hydrogen/src/analytics-manager/AnalyticsProvider.tsx

```diff
@@ -37,6 +37,7 @@ import {ShopifyAnalytics} from './ShopifyAnalytics';
 import {CartAnalytics} from './CartAnalytics';
 import type {CustomerPrivacyApiProps} from '../customer-privacy/ShopifyCustomerPrivacy';
 import type {Storefront} from '../storefront';
+import invariant from 'tiny-invariant';
```

Comment:
Tiny note: we don't use tiny-invariant anywhere else 😬

Reply:
I put the invariant back - Had a team discussion and most agree that it needs to be catastrophic failure if the expected env is not present .. with the ability to opt-out the throw error